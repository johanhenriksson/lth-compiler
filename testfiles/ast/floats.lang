int main() {
    float f;
    f = 3.5;
}
