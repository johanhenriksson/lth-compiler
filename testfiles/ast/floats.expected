Program 
  List 1 item(s)
    Function main()
      IdDecl int main
        TypeId int
      List 0 item(s)
      Block 2 statement(s)
        List 2 item(s)
          DeclareStmt float f;
            IdDecl float f
              TypeId float
            Opt ?
          AssignStmt f = 3.5;
            IdUse f
            FloatLiteral 3.5
